dapp/js/containers/Pages/CampaignDeployer/Component.jsx

```diff
@@ -28,6 +28,16 @@ export default class Deployer extends Component {
 
   componentDidMount() {
     this.props.setAccount(web3.eth.accounts[0]);
+    //set default account to user current account if they are not set
+    let campaignValues = Object.assign({}, this.props.campaignValues);
+    let accountTypes = ['escapeCaller', 'securityGuard', 'donor', 'recipient']
+    for(let i=0; i<accountTypes.length; i++) {
+        if(!campaignValues[accountTypes[i]]){
+            campaignValues[accountTypes[i]] = web3.eth.accounts[0];
+        }
+    }
+    this.props.updateCampaignValues(campaignValues);
```

Comment:
The campaign deployer will last longer than the rest of the dapp, we def need to get this to work and expect it to stay the same until we change the smart contracts... which wont be for at least the next several months :-)